### DISCUSSION TITLE
Where can I find the properties of blocks?

### DISCUSSION THREAD

**Discussion by andromdaa** [Minestom/Minestom#781] [Help] [answered]
ex: ```// Property names are defined by Mojang and usable in various commands
Block tnt = Block.TNT.withProperty("unstable", "true");```

**Comment by TheMode** [comment, marked as answer]
The properties are directly taken from vanilla. 
In your example the block would be comparable to the one placed with `/setblock ~ ~ ~ tnt[unstable=true]`
https://minecraft.fandom.com/wiki/Block_states#TNT
